boards/Makefile.common

```diff
@@ -122,10 +122,10 @@ clean::
 	$(Q)$(CARGO) clean $(VERBOSE)
 
 .PHONY: release
-release: target/$(TARGET)/release/$(PLATFORM).bin
+release: target/$(TARGET)/release/$(PLATFORM).bin target/$(TARGET)/release/$(PLATFORM).elf
```

Comment:
Since we already generate elfs as part of the build process, I think the more idiomatic way to handle this in make would be to mark them as `.PRECIOUS` so that the intermediate isn't deleted rather than adding them as another target. Bonus of this approach is that it'll apply to all elfs without needing to list them out, which is probably what we want anyway.

Reply:
That is much nicer - thanks! I didn't know about `.PRECIOUS`